Source/WebKit/Platform/IPC/haiku/ConnectionHaiku.cpp

```diff
@@ -54,32 +61,73 @@ namespace IPC{
     void Connection::platformInvalidate()
     {
 
+    }
+    void Connection::prepareIncomingMessage(BMessage* message)
+    {
+    	size_t size;
+    	const uint8_t* Buffer;
+    	type_code type;
+    	int32 count;
+    	status_t result;
+    	message->GetInfo("bufferData",&type,&count);
+    	fprintf(stderr,"\n******** %ld ---- ",getpid());
+    	message->PrintToStream();
+    	fprintf(stderr,"*********\n");
+    	if(count == 1)
```

Comment:
I'm not sure what you are trying to do here

Reply:
Sorry I thought reply from another process was getting added to same message turns out it's just print to stream getting mixed up